manifests/i/iTopDataRecovery/iTopDataRecovery/3.0.0.177/iTopDataRecovery.iTopDataRecovery.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: iTopDataRecovery.iTopDataRecovery
+PackageVersion: 3.0.0.177
+Installers:
+- Architecture: neutral
+  InstallerType: inno
+  InstallerUrl: https://goto.itopvpn.com/downloadcenter?product=idr

```

Better to use the direct URL than a redirected one
```suggestion
  InstallerUrl: https://download.itopvpn.com/dl/itop-data-recovery-setup.exe
```